### PR TITLE
Multiple Fasta defined mods in README.md

### PR DESCRIPTION
**Pull request by lutzfischer**

### CODE REVIEW

**Comment by colin-combe** [comment]
"If more then one modification is expected"
should be 
"If more **than** one modification is expected"

File: README.md [hunk 1]
```diff
@@ -360,6 +360,11 @@ Site-specific modifications that are always on (site-specific and fixed) can be
 For a site-specific modification that is not always on (site-specific and variable), the modification is introduced in brackets in the sequence
 
     GRS(ph)KMLN
+    
+If more then one modification is expected to present, these can be defined as a list separated by "|". E.g. modification on a specific Serien can be ph or ac (or none) would be defined as:
```
Comment (lutzfischer) [inline README.md:364]:
```suggestion
If more than one modification is expected to present, these can be defined as a list separated by "|". E.g. modification on a specific Serien can be ph or ac (or none) would be defined as:
```

**Comment by grandrea** [comment]
serine, not serien

**Comment by lutzfischer** [comment]
> serine, not serien

thank you

**Pull request merged**